src/components/DeliverableView/DeliverableStatusBadge.tsx

```diff
@@ -35,14 +37,14 @@ const DeliverableStatusBadge = (props: DeliverableStatusBadgeProps): JSX.Element
           backgroundColor: theme.palette.TwClrBgWarningTertiary,
           borderColor: theme.palette.TwClrBrdrWarning,
           labelColor: theme.palette.TwClrTxtWarning,
-          label: strings.APPROVED,
+          label: strings.IN_REVIEW,
```

Comment:
I hope this change is correct, it is unrelated to the tickets but seemed like a typo so I fixed it up.